azure-pipelines.yml

```diff
@@ -0,0 +1,50 @@
+# .NET Desktop
```

Comment:
Is this one of those continuous integration environments? Should one be setup for the project now that this file is present?

Reply:
Originally I had Windows/Mac/Linux in testing, but WinPcap does not have a silent installation mode, so for this pull request I removed Windows part.

I am still investigating what options can be, so that will come in a separate pull request.

Reply:
Added WinPcap installation via Chocolatey in 
https://github.com/chmorgan/sharppcap/pull/61/commits/f18ab0089f25e9e0a516b657cd787d7bce3a7fe5 so Windows CI is also implemented.

Reply:
Now CI works on all three platforms, https://dev.azure.com/lextudio/lextudio/_build/results?buildId=58